catalog.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
     <link rel="icon" href="favicon.ico">
   </head>
-  <body>
-    <header>
-      <nav>
-        <a href="index.html"><img src="img/logo.svg" width="138" height="70" alt="логотип"></a>          
-        <ul>
+  <body class="body">
+    <header class="main-header">
+      <nav class="main-navigation">
+        <a class="main-header-logo" href="index.html"><img src="img/logo.svg" width="138" height="70" alt="логотип"></a>          
+        <ul class="site-navigation">
           <li><a href="blank.html">Информация</a></li>
           <li><a href="blank.html">Фото и видео</a></li>
           <li><a href="blank.html">Карта штата</a></li>
           <li><a href="catalog.html">Гостиницы</a></li>
         </ul>
       </nav>
     </header>
-    <main>
+    <main class="container">
```

Comment:
Здесь тоже класс нужно убрать. Просто тег без класса.
```suggestion
    <main>
```